### DISCUSSION TITLE
How do i set up this service? I need help with port forwarding please!!!!!!!

### DISCUSSION THREAD

**Discussion by gege42o** [nextcloud/all-in-one#4718] [Questions] [answered]
Hello, so i ran the docker commands and the install script but now i am stuck at the 'input your domain' step. I set up DuckDNS so now I have a domain and installed duckdns on the machine where i'm trying to install nextcloud on as it was said on the install part of the duckdns website. I have port forwarded the 443 and 3478 on my router and have disabled my firewall sine i thought that was blocking the requests but still it says that it cant interact with port 443. The photo attached is the setup on the router page : 
![image](https://github.com/nextcloud/all-in-one/assets/72046316/ee1038b4-337c-4c0f-a9f7-0a49c016fd93)

I do not know what to do, can anyone point me in the right direction? 
I can provide logs or stuff that you request.

**Comment by szaimen** [comment]
Hi, is the public IP an actual public IP or a private ip?

**Reply by gege42o** [reply]
Yes it is a public ip, i got it from whats my ip online to be sure it is the right one

**Reply by szaimen** [reply, marked as answer]
I see. See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-skip-the-domain-validation then
